frontend/pages/beta/model/[modelId]/access-request/schema.tsx

```diff
@@ -27,9 +27,10 @@ export default function NewSchemaSelection() {
     router.push(`/beta/model/${modelId}/access-request/new?schemaId=${newSchema.id}`)
   }
 
-  if (isSchemasError) {
-    return <MessageAlert message={isSchemasError.info.message} severity='error' />
-  }
+  const error = MultipleErrorWrapper(`Unable to fetch schemas`, {
```

Comment:
We have a generic message format of "Unable to load X page". So for here it would be "Unable to load schema page"